slycot/transform.py

```diff
@@ -333,6 +333,254 @@ def tb04ad(n,m,p,A,B,C,D,tol1=0.0,tol2=0.0,ldwork=None):
     kdcoef = max(index)+1
     return A[:Nr,:Nr],B[:Nr,:m],C[:p,:Nr],Nr,index,dcoeff[:porm,:kdcoef],ucoeff[:porm,:porp,:kdcoef]
 
+
+def tb05ad(n, m, p, jomega, A, B, C, job='NG'):
+    """At, Bt, Ct, g_jw, rcond, ev, hinvb = tb05ad_a(n, m, p, jomega, A, B, C):
+    
+    To find the complex frequency response matrix (transfer matrix)
+    G(freq) of the state-space representation (A,B,C) given by
+                                   -1
+       G(freq) = C * ((freq*I - A)  ) * B
+
+    where A, B and C are real N-by-N, N-by-M and P-by-N matrices
+    respectively and freq is a complex scalar.
+
+    Required Arguments
+    ------------------
+
+      n :   integer
+            The number of states, i.e. the order of the state
+            transition matrix A.
+
+      m :   integer
+            The number of inputs, i.e. the number of columns in the
+            matrix B.
+
+      p :   integer
+            The number of outputs, i.e. the number of rows in the
+            matrix C.
+
+      freq  complex
+            The frequency freq at which the frequency response matrix
+            (transfer matrix) is to be evaluated. For continuous time
+            systems, this is j*omega, where omega is the frequency to
+            be evaluated. For discrete time systems,
+            freq = exp(j*omega*Ts)
+
+      A :   double precision array, dimension (n,n).
+            On entry, this array must contain the state transition
+            matrix A.
+
+
+      B :   double precision array, dimension (n,m).
+            On entry, this array must contain the input/state matrix B.
+
+
+      C :   double precision array, dimension (p,n)
+            On entry, of this array must contain the state/output matrix C.
+
+
+      job : string, 'AG', 'NG', or 'NH'
+            If job = 'AG' ('all', 'general matrix', the A matrix is
+            first balanced via lapack routine DGEBAL with an orthoganal
```

Comment:
Good point. It's not like I'm documenting every other lapack function that gets called.